remoteworkitem/remoteworkitem.go

```diff
@@ -0,0 +1,73 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+
+	"github.com/almighty/almighty-core/app"
+)
+
+const (
+	SystemRemoteIssueId = "system.remote_issue_id"
```

Comment:
The word "issue" is not commonly used by all remote trackers. I think we can replace it with "item".